vue-app/src/App.vue

```diff
@@ -96,6 +103,28 @@ export default class App extends Vue {
     await this.$store.dispatch(LOAD_RECIPIENT_REGISTRY_INFO)
   }
 
+  beforeDestroy() {
+    for (const interval of Object.keys(this.intervals)) {
+      clearInterval(this.intervals[interval])
+    }
+  }
+
+  @Watch('$web3.user')
```

Comment:
So is this how class based components adds watchers?

Reply:
https://github.com/kaorun343/vue-property-decorator#Watch 👍🏻